src/components/SignalCard/controls/CopySignal.js

```diff
@@ -41,22 +58,29 @@ const CopySignal = ({
     }
   })
 
-  const [isCreation, setCreation] = useState(false)
-
-  return (
-    !isCreation && (
-      <>
-        {children}
-        <Button
-          onClick={copySignal}
-          as={as}
-          className={cx(styles.copyBtn, classes.copyBtn)}
-          {...btnParams}
-        >
-          {label}
-        </Button>
-      </>
-    )
+  return !isCreation ? (
+    <>
+      {children}
+      <Button
+        onClick={copySignal}
+        as={as}
+        className={cx(styles.copyBtn, classes.copyBtn)}
+        {...btnParams}
+      >
+        {label}
+      </Button>
+    </>
+  ) : (
+    <>
+      {children}
+      <Button
+        as={as}
+        className={cx(styles.copyBtn, classes.copyBtn)}
+        {...btnParams}
+      >
+        {label}
+      </Button>
+    </>
```

Comment:
Both operands are identical except the `onClick` property. We can decrease code duplication by using ternary operator only for `onClick` property.